analysed-packages/attr/version-23.1.0/attr-23.1.0-OSS-disclosure.txt

```diff
@@ -0,0 +1,150 @@
+FileChecksum: SHA256: f2507f4b9485817d8ab1f964b0af19ca093a5c35d3a1062d2259a0777ec8f5a6
+LicenseConcluded: MIT
+
+--------
+FileName: site-packages_attr-with-distinfo.zip/site-packages_attr-with-distinfo/attr/setters.py:
+FileChecksum: SHA256: a5b09943ea44e99c630ea65f59652150579f5eda5e908538a92fd80cc2cf439d
+LicenseConcluded: MIT
+
+--------
+FileName: site-packages_attr-with-distinfo.zip/site-packages_attr-with-distinfo/attr/validators.py:
+FileChecksum: SHA256: 0b6310817eee6cbfdcb3963389b59af26d18c5d32ae7fdc2877755233b0b3be6
+LicenseConcluded: MIT
+
+--------
+FileName: site-packages_attr-with-distinfo.zip/site-packages_attr-with-distinfo/attrs/exceptions.py:
+FileChecksum: SHA256: 4a50e58ba018efb7fa9f2f87ee8cbdf0e9108ecaf0f83fecba912e12b2156241
+LicenseConcluded: MIT
+
+--------
+FileName: site-packages_attr-with-distinfo.zip/site-packages_attr-with-distinfo/attrs/filters.py:
+FileChecksum: SHA256: 75cfdd35ecb6f641fa28b535993d836a4abbb59de405fcc41b0cce983cf0d45f
+LicenseConcluded: MIT
+
+--------
+FileName: site-packages_attr-with-distinfo.zip/site-packages_attr-with-distinfo/attrs/__init__.py:
+FileChecksum: SHA256: f7fe706956c5b3bacbaad5d9ef7b4d0ebc617b3c99f1565e5f805bbd0dc4789c
+LicenseConcluded: MIT
+
+--------
+FileName: site-packages_attr-with-distinfo.zip/site-packages_attr-with-distinfo/attrs/setters.py:
+FileChecksum: SHA256: a0ac39d42ef61e1e39c13c18bc31c93fd91b89cc62321311e18e46bdda4a7474
+LicenseConcluded: MIT
+
+--------
+FileName: site-packages_attr-with-distinfo.zip/site-packages_attr-with-distinfo/attrs/validators.py:
+FileChecksum: SHA256: e1a8354b2543d879b73d82a2346fcd3ad47f7bb7fcd47afa1a2365e18bd7a384
+LicenseConcluded: MIT
+
+--------
+FileName: site-packages_attr-with-distinfo.zip/site-packages_attr-with-distinfo/attrs-23.1.0.dist-info/licenses/LICENSE:
+FileChecksum: SHA256: 882115c95dfc2af1eeb6714f8ec6d5cbcabf667caff8729f42420da63f714e9f
+FileCopyrightText:
+Copyright (c) 2015 Hynek Schlawack and the attrs contributors
+
+LicenseConcluded: MIT
+
+--------
+FileName: site-packages_attr-with-distinfo.zip/site-packages_attr-with-distinfo/attrs-23.1.0.dist-info/METADATA:
+FileChecksum: SHA256: ca0970517928ef943e209e8b98f550e18f7d2894b708f2b4356f28bd7158b038
+LicenseConcluded: MIT
+
+--------
+FileName: site-packages_attr-with-distinfo.zip/site-packages_attr-with-distinfo/attr/_version_info.py:
+FileChecksum: SHA256: 7b14aa6f76f913e7cc4ac819025130f5770ba448286cf391099a5c68482500ce
+LicenseConcluded: LicenseRef-fossology-MIT-3d930e873df38b22e2f1db27007eec50
+
+--------
+FileName: site-packages_attr-with-distinfo.zip/site-packages_attr-with-distinfo/attrs/converters.py:
+FileChecksum: SHA256: 7c2044765616726237b029e9524da9cf6d8662d5f3a93929e8d78ea5d23ae0f6
+LicenseConcluded: LicenseRef-fossology-MIT-cf5b9f6aa02362de11792252d20b1a76
+
+--------
+
+
+Referenced licenses:
+--------------------
+
+LicenseRef-fossology-MIT-3d930e873df38b22e2f1db27007eec50:

```

Comment:
I think you wanted to conclude this license for the source files which carry:

> SPDX-License-Identifier: MIT

and not the reference MIT license available in the SPDX license list